### DISCUSSION TITLE
Alignment

### DISCUSSION THREAD

**Discussion by Conaclos** [jgm/djot#205] [General]
Hi!

Is there any way to center / right-align a text or image?

Thanks!

**Comment by jgm** [comment]
Not a built-in way.  Of course, you can add a class to the image and add some CSS that does whatever.

**Comment by uvtc** [comment]
Are you open to adding center-align syntax to djot? (I realize this was discussed ad-infinitum in the Pandoc issue.)

**Reply by Omikhleia** [reply]
What's wrong using classes:

```
{.center}
Some centered text
```

Or "à la Pandoc docx" way (with appropriate style defined in a ref. document)
```
{custom-style=Centered}
Some styled text
```

Why would any other presentation syntax be needed?

**Reply by uvtc** [reply]
Looks good. Thanks.
